pkgs/development/python-modules/joblib/default.nix

```diff
@@ -10,21 +11,31 @@
 
 buildPythonPackage rec {
   pname = "joblib";
-  version = "0.12.4";
+  version = "0.13.2";
 
   # get full repository inorder to run tests
   src = fetchFromGitHub {
     owner = "joblib";
     repo = pname;
     rev = version;
-    sha256 = "06zszgp7wpa4jr554wkk6kkigp4k9n5ad5h08i6w9qih963rlimb";
+    sha256 = "11bspmm2is8akcld2kh6bgp0in1ggdy91qsk5zqybzgapc8b4zdf";
   };
 
+  patches = [
+    # python-lz4 compatibility
+    (fetchpatch {
+      url = "https://github.com/joblib/joblib/pull/847.patch";
+      sha256 = "0b381d1jzbn6ymj9r859ijk4yk9sd00giv51nqwzbbcmpv17kdas";
+    })
+  ];
+
   checkInputs = [ sphinx numpydoc pytest ];
   propagatedBuildInputs = [ python-lz4 ];
 
+  # test_disk_used is broken
```

Comment:
I don't see any failure of this test over at https://github.com/NixOS/nixpkgs/pull/57906. Are you sure it needs to be disabled?

Reply:
It fails on some on my system in the sanbox, on others it does not.